Samples/CameraSettingsExternalSettingsApp/DefaultControlHelper/DefaultControlHelper.h

```diff
@@ -24,77 +24,71 @@ namespace winrt::DefaultControlHelper::implementation
 
     private:
 
-        wil::com_ptr_t<IMFCameraConfigurationManager>           m_configManager;
-        wil::com_ptr_t<IMFCameraControlDefaultsCollection>      m_controlDefaultsCollection;
+        wil::com_ptr_t<IMFCameraConfigurationManager> m_configManager;
+        wil::com_ptr_t<IMFCameraControlDefaultsCollection> m_controlDefaultsCollection;
     };
 
     struct DefaultController : DefaultControllerT<DefaultController>
     {
-        DefaultController(winrt::DefaultControlHelper::DefaultControllerType type, uint32_t id, winrt::weak_ref<DefaultControlManager> manager);
-
-        uint32_t DefaultValue();
-        void DefaultValue(uint32_t const& value);
+        DefaultController(winrt::DefaultControlHelper::DefaultControllerType type, uint32_t id, winrt::com_ptr<DefaultControlManager> manager);
+        bool HasDefaultValueStored() { return m_hasDefaultValueStored; }
+        int32_t DefaultValue();
+        void DefaultValue(int32_t const& value);
 
     private:
 
-        uint32_t                                        m_defaultValue = 0;
-        std::optional<MF_CAMERA_CONTROL_RANGE_INFO>     m_rangeInfo;
-        wil::com_ptr<IMFCameraControlDefaults>          m_controlDefaults;
+        bool m_hasDefaultValueStored;

```

Comment:
nit: init here

Reply:
I know, but this class/header already does default inits here, so lets keep it in one place.